### PR TITLE
Add eval & prod profiles for observability

### PR DESCRIPTION
**Pull request by clebs**
**Description**

Changes proposed in this pull request:

- Add evaluation and production profiles for observability components.

**Related issue(s)**
#9689

### CODE REVIEW

**Review by suleymanakbas91 — APPROVED** [review]

File: resources/monitoring/profile-production.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+prometheus:
+  prometheusSpec:
+    resources:
+      limits:
+        cpu: 600m
+        memory: 1500Mi
```
Comment (a-thaler) [inline resources/monitoring/profile-production.yaml:6]: Isn't there a mistake?

File: resources/monitoring/profile-production.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        cpu: 150m
+        memory: 1000Mi
+    retention: 30d
+    retentionSize: 15GB
+    storageSpec:
+      volumeClaimTemplate:
+        spec:
+          resources:
+            requests:
+              storage: 20Gi
```
Comment (a-thaler) [inline resources/monitoring/profile-production.yaml:17]: Shouldn't we reduce the size heavily on evaluation profile?

Comment (clebs) [reply resources/monitoring/profile-production.yaml:17]: that is the production profile. This was copied from:
https://github.com/kyma-project/kyma/blob/19a922844fe8b30d8f251f33dabdebed1bbd75c1/installation/resources/installer-config-production.yaml.tpl#L67

File: resources/monitoring/profile-production.yaml (hunk 1 above)
Comment (clebs) [reply resources/monitoring/profile-production.yaml:6]: True, should be 3Gi, will fix it.

File: resources/monitoring/profile-production.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+prometheus:
+  prometheusSpec:
+    resources:
+      limits:
+        cpu: 600m
```
Comment (a-thaler) [inline resources/monitoring/profile-production.yaml:5]: CPU requests and limits are also not fitting, see: https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-config-production.yaml.tpl

Comment (clebs) [reply resources/monitoring/profile-production.yaml:5]: Fixed thanks

**Pull request merged**